renderer.js

```diff
@@ -985,14 +1010,84 @@ function playPressed(){
 
         playModalQuitDLCancel = dataObj.reqObj;
         
+        
     });
 
 }
 
-playButtonText.addEventListener("click", function(event){
+async function checkIfCompatible() {
+    try {
+        const data = await si.graphics();
+
+        // Get all of the graphics cards info
+        var i;
+        for (i = 0; i < data.controllers.length; i++){
+            graphicControllers = data.controllers[i];
+            cards = String(graphicControllers.model);
+            console.log(cards);
+        }
+
+        // Is incompatible if the word Intel is found in a substring
+        if(cards.includes("Intel")){
```

Comment:
This does not seem to work. I tested on my laptop that has nvidia and intel graphics, and it did not give me any kind of popup.